113. Path Sum II

```diff
@@ -0,0 +1,46 @@
+Time complexity: O(Max depth)
```

Comment:
So the leetcode explanation says that the time complexity is O(n^2) and in the comments people say it os O(nlogn). Copying the list is O(m) operation will be logn. This list can be the height of the tree at max. This is my understanding. Worst case it will O(n^2).